api/routes/userRouter.js

```diff
@@ -1,19 +1,16 @@
 const express = require('express');
 const router = express.Router();
-const userController = require('../controllers/userController.js');
-const userSchema = require('../schemas/userSchema.js');
+const userController = require('../controllers/userController');
+const userSchema = require('../schemas/userSchema');
 const validator = require('express-joi-validation').createValidator({});
 
 router.route('/')
     .post(validator.body(userSchema.schema), userController.create)
-    .get(userController.findAll);
-
-router.route('/suggest')
-    .get(userController.suggestUsers);
+    .get(userController.getAll);
```

Comment:
Just a small note, we can also use joi validations for query and path params.

Reply:
thank for recommendation, joi will be used for query and path params in next tasks.